index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Mukta:wght@200&display=swap" rel="stylesheet">
+    <title>Old Apple Clone</title>
+</head>
+<body>
+    <main class="grid-container">
+        <div class="header">
+          <div class="big-image">
+
+              <nav class="navbar">
+                  <div class="navbar-item"><a href="#"><img class="apple-logo" src="./images/apple.png" alt="apple logo"></a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">Store</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">Mac</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">iPod</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">iPhone</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">iPad</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">iTunes</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">Support</a></div>
+                  <div class="search-bar">
+                    <img class="search-icon" src="./images/search.svg" alt="search icon" />
+                    <input type="search" name="search" id="search">
+                  </div>
+              </nav>
+            <div class="header-text">
+                <p class="big-text">What will your verse be?</p>
+                <p class="medium-text">Everyone has something to share. See how poeple are using iPad in their unique ways to add world's story.</p>
+                <p class="small-text"><a href="#">Watch the "Your Verse" film and explore the stories</a> <i class="fa fa-play-circle-o" aria-hidden="true"></i>  </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="photo1">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/iphone1.jpg" alt="Iphone 5s"></a>
+          </div>
+        </div>
+        <div class="photo2">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/iphone2.jpg" alt="Iphone 5c"></a>
+          </div>
+        </div>
+        <div class="photo3">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/apple1.jpg" alt="Mac birthday"></a>
+          </div>
+        </div>
+        <div class="photo4">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/apple2.jpg" alt="Supplier Responsability"></a>
+          </div>
+        </div>
```

Comment:
## Add `background-image` or `background: url()` CSS property

- The main content should have an image set as background (as CSS [`background-image`](https://www.w3schools.com/cssref/pr_background-image.asp) or [`background -> url`](https://www.w3schools.com/cssref/css3_pr_background.asp)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_image_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients))

- [ ]  Remove all your `<img>` tags in this section
- [ ] Add a id on each `<a>` tag
- [ ] In your CSS file add `background-image`  property on each id.

Reply:
On the original page, they are using <img> tags to implement the document I don't understand the need for this requirement. The background image is used in the header cell where it is needed as in the original page
![image](https://user-images.githubusercontent.com/66382067/87959801-0f041200-ca79-11ea-8b85-914efeae512b.png)